core/src/main/kotlin/com/gitlab/kordlib/core/cache/data/PartialGuildData.kt

```diff
@@ -0,0 +1,22 @@
+package com.gitlab.kordlib.core.cache.data
+
+import com.gitlab.kordlib.cache.api.data.description
+import com.gitlab.kordlib.common.entity.Permissions
+import kotlinx.serialization.Serializable
+
+@Serializable
+class PartialGuildData(
+        val id: Long,
+        val name: String,
+        val icon: String? = null,
+        val owner: Boolean? = null,
+        val permissions: Permissions? = null
+) {
+    companion object {
+        val description = description(PartialGuildData::id) {
```

Comment:
Do we plan on storing PartialGuilds in cache?

Reply:
We don't receive that from the gateway I made a mistake.